src/main/kotlin/no/nav/faktureringskomponenten/featuretoggle/LocalUnleash.kt

```diff
@@ -0,0 +1,122 @@
+package no.nav.melosys.featuretoggle
```

Comment:
package melosys? Her er det også ubrukte commits, og egentlig for meg ser det ut som en del ubrukt kode også. Virker å være ren copy/paste fra API, men dere bør være litt forsiktig med det. 

Lag gjerne en PR med opprydding av dette.